data/formatters/generic.yaml

```diff
@@ -14,6 +14,34 @@ short_message:
 - '{http_request}'
 - 'from: {ip_address}'
 ---
+type: 'conditional'
+data_type: 'aws:elb:access'
+message:
+- 'type: {type}'
+- 'resource_identifier: {resource_identifier}'
+- 'client_ip_address: {client_ip_address}'
```

Comment:
I opt to use `source / destination` IP or (`client / server` if you have to) but I opt not to mix terminology pairs

Reply:
This has been changed to source / destination